test_util/test_upgrade_vpc.py

```diff
@@ -115,6 +115,9 @@ def main():
     stable_installer_url = os.environ['STABLE_INSTALLER_URL']
     installer_url = os.environ['INSTALLER_URL']
 
+    config_yaml_override_install = os.getenv('CONFIG_YAML_OVERRIDE_INSTALL')
+    config_yaml_override_upgrade = os.getenv('CONFIG_YAML_OVERRIDE_UPGRADE')
```

Comment:
There's a default config that these configs get merged with, so it might be helpful to someone who's debugging a failed job if there's a hint somewhere that these aren't the final configs that get used. No opinion on whether `OVERRIDE` is the right word to communicate that.